aslam_offline_calibration/kalibr/python/kalibr_camera_calibration/MulticamGraph.py

```diff
@@ -9,7 +9,7 @@
 import itertools
 import sys
 import pylab as pl
-import Image
+from PIL import Image
```

Comment:
@floriantschopp , do you remember why you did this? What was the issue with the import before?

Reply:
On 18.04 I think you need this, otherwise you get an import error that there is no module named Image.

Reply:
Okay, thanks. I've just added a few lines for backwards compatibility.